server/src/index.js

```diff
@@ -23,26 +24,29 @@ app.use(compression());
 app.use(helmet());
 
 // cors enable
-app.options('*', cors());
-app.use(cors({ origin: 'http://localhost:5000' }));
+app.options("*", cors());
+app.use(cors({ origin: "http://localhost:5000" }));
 
 // body-parser
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
 // database connection
-mongoose.connect(process.env.MONGO_URL,
+mongoose
+  .connect(
+    `mongodb+srv://${process.env.MONGO_DB_USER}:${process.env.MONGO_DB_PASSWORD}@kurakoo.4ebj8.mongodb.net/${process.env.MONGO_DB_DATABASE}?retryWrites=true&w=majority`,
     {
-        useNewUrlParser: true,
-        useUnifiedTopology: true,
-        useCreateIndex: true
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useCreateIndex: true,
     }
-).then(()=>{
+  )
+  .then(() => {
     console.log("database connected");
-});
+  });
 
 // all the api routes
-app.use('/api', root);
+app.use("/api", authRoutes);
```

Comment:
I would recommend you to read the structure of the api properly first and then change the things, like the root will return to here always and no other route will come here. The routes like authRoutes which you have created will go on to the `root.js` file in the routes directory, as done for `example.route.js` which is there for reference which you have deleted and must not be done at the moment.

Reply:
This too